pkg/services/dashboards/service/dashboard_service.go

```diff
@@ -1556,6 +1550,19 @@ func (dr *DashboardServiceImpl) UnstructuredToLegacyDashboard(ctx context.Contex
 	return &out, nil
 }
 
+func (dr *DashboardServiceImpl) getUser(ctx context.Context, userMeta string) (*user.User, error) {
+	if userMeta == "" || toUID(userMeta) == "" {
+		// TODO: will there always be a creator?  ( should we error when creator is missing? )
+		return &user.User{}, nil
+	}
+	uid := toUID(userMeta)
+	userId, err := strconv.ParseInt(uid, 10, 64)
+	if err == nil {
+		return dr.userService.GetByID(ctx, &user.GetUserByIDQuery{ID: userId})
+	}
+	return dr.userService.GetByUID(ctx, &user.GetUserByUIDQuery{UID: uid})
```

Comment:
Could we add a check if the error is `ErrUserNotFound`, and if so just return an empty user, and no error?

Reply:
@stephaniehingtgen updated